docs/source/en/task_summary.mdx

```diff
@@ -73,7 +80,17 @@ Image classification labels an entire image from a predefined set of classes. Li
 >>> from transformers import pipeline
 
 >>> classifier = pipeline(task="image-classification")
->>> classifier("path/to/image/file.jpg")
+>>> preds = classifier(
+...     "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"
+... )
+>>> preds = [{"score": round(pred["score"], 4), "label": pred["label"]} for pred in preds]
+>>> preds
+[{'score': 0.4403, 'label': 'lynx, catamount'},
```

Comment:
Is this formatted by `make style`? I feel we can have a better representation. See comments below too.